src/Omnipay/WorldPayXML/Message/Response.php

```diff
@@ -12,13 +12,21 @@
  */
 class Response extends AbstractResponse
 {
+    /** @var string */
+    protected static $PAYMENT_STATUS_AUTHORISED             = 'AUTHORISED';
```

Comment:
Should these not be constants?

Reply:
Yep probably. I don't think they worked in the other repo with the Trait & these were the closest functional equivalent. Will constant-ify in this repo for now.